packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -82,6 +82,8 @@ export const messages = {
     'Deleting source files deletes the files from your computer and removes the corresponding metadata from your default org. Are you sure you want to delete this source from your project and your org?',
   confirm_delete_source_button_text: 'Delete Source',
   cancel_delete_source_button_text: 'Cancel',
+  select_manifest_file_button_text: 'Select Manifest File',
+  select_manifest_filter_button_text: 'Manifest',
```

Comment:
@ruthemmanuelle both of these strings would need your approval (or correction)

Reply:
@JimSuplizio - These look reasonable. In your screenshot, it looks like Manifest is an option in a dropdown menu. Are there other options in that menu?

Reply:
@ruthemmanuelle No, there are no other options, the only valid type of manifest file is an XML file. In VS Code filters are always a dropdownlist which, in this case, translates to a dropdownlist that only has 1 item.

Reply:
Ok. I'll approve the PR.